content/integrations/grafana.md

```diff
@@ -21,7 +21,7 @@ Verify that you can access grafana by accessing it using a browser: http://local
 Install the iter8 dashboard using the provided install command:
 
 ```bash
-curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8/{{< versionNumber >}}/integrations/grafana/install_dashboard.sh \
+curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8/{{< versionNumber >}}/hack/grafana_install_dashboard.sh \
```

Comment:
This link is broken

Reply:
https://raw.githubusercontent.com/iter8-tools/iter8/v1.0.0-rc3/hack/grafana_install_dashboard.sh